src/_includes/partials/footer.html

```diff
@@ -0,0 +1,54 @@
+<footer>
+	<div>© 2021 lunch.dev</div>
+
+	<div class="footer-links">
+		<a href="/code-of-conduct">Code of Conduct</a>
+		<span class="footer-divider">•</span>
+		<a href="/">Events</a>
+		<span class="footer-divider">•</span>
+		<a href="/sesh">Sesh</a>
+	</div>
+
+	<div class="footer-social">
+		<a href="https://discord.com/invite/q9ZdBgP" target="_blank" rel="noreferrer noopener">
```

Comment:
We should add an `aria-label` so screenreader users tabbing to this link know where it's going.

```suggestion
		<a href="https://discord.com/invite/q9ZdBgP" target="_blank" rel="noreferrer noopener" aria-label="Discord">
```